hooks/useLobbyActions.ts

```diff
@@ -44,8 +44,19 @@ export default function useLobbyActions() {
   }
 
   function onStartGame() {
-    const status: RoomStatus = "waiting";
-    update(roomRef, { status });
+    runTransaction(teamsRef, (teams: Team[]) => {
+      if (teams.length < 2) canBeStarted = false;
+      teams.forEach((team, index) => {
+        if (index != 0 && team.members?.length < 2) canBeStarted = false;
+        if (index != 0) numberOfTotalTeamPlayers += team.members?.length;
+        if (numberOfTotalTeamPlayers > 12) canBeStarted = false;
+      });
+      return teams;
+    });
+    if (canBeStarted) {
+      const status: RoomStatus = "waiting";
+      update(roomRef, { status });
+    }
```

Comment:
maybe add an `alert` for the reason why the game can't start? Just a simple alert nothing fancy needed yet.